dist/images/Dockerfile

```diff
@@ -1,44 +1,54 @@
 #
-# This is the OpenShift sdn-ovn image.
-# it provides an overlay network using ovs/ovn/ovnkube
+# This is the OpenShift ovn overlay network image.
+# it provides an overlay network using ovs/ovn/ovn-kube
 #
-# The standard name for this image is origin-network-ovn
+# The standard name for this image is ovn-kube
 
 # Notes:
-# The ovn-kubernetes utilities can come from the openvswitch-ovn-kubernetes rpm
-# or built in go_controller, copied to this directory and copied to the container.
-# The question becomes one of determining which version are we using.
+# This is for a development build where the ovn-kubernetes utilities
+# are built locally and included in the image (instead of the rpm)
 #
-# At present this uses fedora:28, this may default to centos:7 
-# when the needed ovs/ovn rpms are at 2.8 or higher and the
-# openvswitch-ovn-kubernetes is available.
+# At present this uses centos:7
+# openvswitch rpms are from
+# http://cbs.centos.org/kojifiles/packages/openvswitch/2.9.0/4.el7/x86_64/
 #
 # So this file will change over time.
 
-FROM fedora:28
-
-MAINTAINER Phil Cameron <pcameron@redhat.com>
+FROM centos:7
 
 USER root
 
 ENV PYTHONDONTWRITEBYTECODE yes
 
-RUN dnf install --refresh -y  \
-	PyYAML bind-utils procps-ng hostname \
-	openvswitch openvswitch-ovn-common openvswitch-ovn-host openvswitch-ovn-central \
-	openvswitch-ovn-docker python2-openvswitch openvswitch-ovn-vtep \
-	containernetworking-cni \
-	jq iproute strace socat && \
-	dnf clean all
+RUN yum install -y  \
+	PyYAML bind-utils \
+	libnuma.so \
+	openssl \
+	numactl-libs \
+	firewalld-filesystem \
+	libpcap \
+	iproute strace socat && \
+	yum clean all
+
+# get a reasonable version of openvswitch
+RUN rpm -i http://cbs.centos.org/kojifiles/packages/openvswitch/2.9.0/4.el7/x86_64/openvswitch-2.9.0-4.el7.x86_64.rpm
```

Comment:
@pecameron For images built for OCP, you *cannot* pull RPMs from external sources. They *will* fail QE security checks. All RPMs must come from internal, vetted sources.

Reply:
@adammhaile This set of changes builds the community image, not the OCP image. I thought the OCP image was going to be built in the dist-git repo that was recently created. I will need some help in getting a proper Dockerfile for the OCP image.

Reply:
@pecameron Ah, right... that will be from http://github.com/openshift/ose-ovn-kubernetes
Sorry for confusion. We will pull the dockerfile and related files from that repo and put them into the dist-git then build from there. Let me know when you need help with the OCP version of the dockerfile.